tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/model/Type.java

```diff
@@ -22,7 +22,15 @@
  */
 public enum  Type {
     /**
-     * Hawkular APM environment (requires instrumentation of the application).
+     * Hawkular APM Agent environment (requires instrumentation of the application using java agent).
```

Comment:
Can we change this to something like: will instrument application with Hawkular Java agent?